crypto/sha/asm/keccak1600-armv4.pl

```diff
@@ -933,7 +933,7 @@ sub Round {
 ___
 }
 
-{ my ($out,$len,$A_flat,$bsz,$next) = map("r$_", (4,5,10,12,11));
+{ my ($out,$len,$A_flat,$bsz,$next) = map("r$_", (4,5,10,12,0));
```

Comment:
It seems reasonable to use r0. 
It is used down on line 984 as temp, so this should be ok.